### ISSUE TITLE
Translation through files in ROM

### ISSUE THREAD

**Issue by adrifcastr** [Zarel/Pokemon-Showdown#3594]
The ROMS for example sumo, contain all the translated stuff. It would be so nice to finally have this translated. I can help in providing needed files.

**Comment by Zarel** [comment]
Yes, that's the plan.

For some reason I can't seem to find an older translation issue...

**Comment by adrifcastr** [comment]
I already made one like in 2016 or something, btw I think the "beta" is underrating showdown, showdown is excellent, so I wouldn't call it beta, but there's your choice, if you need any help regarding the text files etc... I can help out with that, I'd say I am a pretty advanced rom hacker, and I'd also love to have showdown in my native languages, so I am also willing to help to contribute to make this happen.

**Comment by Zarel** [comment]
Giving me the files in any easily parsable format would be a good step.

I want to get offline messaging and friend lists done first, but after that, this is next.

**Comment by adrifcastr** [comment]
okay that sounds great (does the friend system means the'll be a multi battle, battle royale and stuff like that?)  And yeah, I'll talk to KazoWAR and see, what he says on how we'll do this, but I believe I can Export most of the Text-Stuff to .txt

**Comment by adrifcastr** [comment]
@Zarel I finally had time to dump the text! 
[ShowdownTXT.zip](https://github.com/Zarel/Pokemon-Showdown/files/1220104/ShowdownTXT.zip)

**Comment by Vegalius** [comment]
> 
> 
> Giving me the files in any easily parsable format would be a good step.
> 
> I want to get offline messaging and friend lists done first, but after that, this is next.

@Zarel I'm probably a bit late to the party, but is there a status on the Friends List feature? If so, who's working on that? If not, I was wondering if I could tackle that.

**Comment by Zarel** [comment]
@Vegalius Friends list is dependent on the Preact rewrite.

**Comment by Vegalius** [comment]
@Zarel Yeah, @AustinXII was telling me the same as it was delayed because of the rewrite, in which I imagine WILL definitely take time to complete.

**Comment by Zakutsu** [comment]
hey @Zarel , I'm more than late, but how is it that this is still not translated? as long as I know at least friend lists are already done, do you need any help with it?

**Comment by Zarel** [comment]
We are happy to accept translation PRs, but no one has sent any in.
